ztp/source-crs/HardwareEventSubscriptionNS.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-hw-events
+  annotations:
+    workload.openshift.io/allowed: management
```

Comment:
This annotation should only be included on operators that are part of the OCP base platform. Please remove it from here so that the functionality under the hw events operator is accounted for under the non-ocp platform CPU budget.
```suggestion
```

Reply:
updated